src/lib.rs

```diff
@@ -172,12 +191,25 @@ pub async fn main(req: Request, env: Env, _ctx: Context) -> Result<Response> {
                                         {
                                             Ok(_) => {
                                                 console_log!("saved published note: {}", event.id);
+                                                let relay_msg =
+                                                    RelayMessage::new_ok(event.id, true, "");
+                                                server
+                                                    .send_with_str(&relay_msg.as_json())
+                                                    .expect("failed to send response");
                                             }
                                             Err(e) => {
                                                 console_log!(
                                                     "could not save published note: {} - {e:?}",
                                                     event.id
                                                 );
+                                                let relay_msg = RelayMessage::new_ok(
+                                                    event.id,
+                                                    false,
+                                                    "error: could not save published note",
+                                                );
+                                                server
+                                                    .send_with_str(&relay_msg.as_json())
+                                                    .expect("failed to send response");
```

Comment:
for these not sure if I am supposed to be using `server.send_with_str` or `pair.client.send_with_str`

Reply:
seems to work